clang/lib/Sema/SemaTemplateDeductionGuide.cpp

```diff
@@ -1072,12 +1072,27 @@ BuildDeductionGuideForTypeAlias(Sema &SemaRef,
       AliasRhsTemplateArgs, TDeduceInfo, DeduceResults,
       /*NumberOfArgumentsMustMatch=*/false);
 
+  static std::function<bool(const TemplateArgument &TA)> IsNonDeducedArgument =
```

Comment:
I  meant a function in the file, not a std::function! (that would be...inefficient haha)